code/src/java/pcgen/gui2/facade/Gui2InfoFactory.java

```diff
@@ -125,15 +126,13 @@
  * The Class {@code Gui2InfoFactory} provides character related information
  * on various facade objects. The information is displayed to the user via the 
  * new user interface. 
- *
- * 
  */
 public class Gui2InfoFactory implements InfoFactory
 {
 	/** A default return value for an invalid request. */
 	private static final String EMPTY_STRING = ""; //$NON-NLS-1$
-	private static final NumberFormat ADJ_FMT = new DecimalFormat("+0;-0"); //$NON-NLS-1$
-	private static final NumberFormat COST_FMT = new DecimalFormat("#,##0.##"); //$NON-NLS-1$
+	private final Format ADJ_FMT = new DecimalFormat("+0;-0"); //$NON-NLS-1$
```

Comment:
I still don't understand the value of this change.  If it is thrown around as merely a Format, we lose methods on the interface and it decreases the amount of semantic information available to the developer.  I see these as negative value changes.  YMMV.